Makefile

```diff
@@ -23,7 +23,7 @@ install: clean ## Creates venv, and adds biomage as system command
 
 	@echo "==> Installing utility and dependencies..."
 	@venv/bin/pip install --upgrade pip
-	@venv/bin/pip install .
+	@venv/bin/pip install -e .
```

Comment:
What does the `-e` flag do?

Reply:
Installs with symlinks; it means you can develop utils without reinstalling after my recent packaging changes.

Reply:
it links the installed packages to the source code so you don't have you `make install` when you develop in order to make the changes available for `biomage` command.